src/core/importType.js

```diff
@@ -29,7 +29,11 @@ export function isBuiltIn(name, settings) {
 
 function isExternalPath(path, name, settings) {
   const folders = (settings && settings['import/external-module-folders']) || ['node_modules']
-  return !path || folders.some(folder => -1 < path.indexOf(join(folder, name)))
+
+  // extract the part before the first / (redux-saga/effects => redux-saga)
```

Comment:
what about scoped packages? Let's also add a test for `@foo/bar` or something.

Reply:
Done!